iothub_client/src/iothub_module_client_ll.c

```diff
@@ -246,6 +246,36 @@ IOTHUB_CLIENT_RESULT IoTHubModuleClient_LL_GetTwinAsync(IOTHUB_MODULE_CLIENT_LL_
     return result;
 }
 
+IOTHUB_CLIENT_RESULT IoTHubModuleClient_LL_GetTwinDesiredAsync(IOTHUB_MODULE_CLIENT_LL_HANDLE iotHubModuleClientHandle, IOTHUB_CLIENT_GET_TWIN_REQUEST_OPTIONS* twinRequestOptions, IOTHUB_CLIENT_DEVICE_TWIN_SECTION_CALLBACK moduleTwinDesiredCallback, void* userContextCallback)
+{
+    IOTHUB_CLIENT_RESULT result;
+    if (iotHubModuleClientHandle != NULL)
+    {
+        result = IoTHubClientCore_LL_GetTwinDesiredAsync(iotHubModuleClientHandle->coreHandle, twinRequestOptions, moduleTwinDesiredCallback, userContextCallback);
+    }
+    else
+    {
+        LogError("Input parameter cannot be NULL");
+        result = IOTHUB_CLIENT_INVALID_ARG;
+    }
+    return result;
```

Comment:
nit: spacing between logical operation blocks.
```suggestion
    IOTHUB_CLIENT_RESULT result;
    
    if (iotHubModuleClientHandle != NULL)
    {
        result = IoTHubClientCore_LL_GetTwinDesiredAsync(iotHubModuleClientHandle->coreHandle, twinRequestOptions, moduleTwinDesiredCallback, userContextCallback);
    }
    else
    {
        LogError("Input parameter cannot be NULL");
        result = IOTHUB_CLIENT_INVALID_ARG;
    }
    
    return result;
```

Reply:
In this case I'm just keeping with the existing pattern for all the funtions.